i18n/nl.json

```diff
@@ -152,13 +143,13 @@
     "globals.buttons.saveChanges": "Veranderingen opslaan",
     "globals.buttons.view": "Bekijken",
     "globals.days.0": "Zo",
-    "globals.days.1": "Zo",
-    "globals.days.2": "Ma",
-    "globals.days.3": "Di",
-    "globals.days.4": "Wo",
-    "globals.days.5": "Do",
-    "globals.days.6": "Vr",
-    "globals.days.7": "Za",
+    "globals.days.1": "Ma",
```

Comment:
Could you please confirm that changing the beginning of the work doesn't break date displays and the campaign scheduling calendar?